### PR TITLE
BrokerService_GetCurrency

### PR DESCRIPTION
**Pull request by denis-kriv**

### CODE REVIEW

File: Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs [hunk 1]
```diff
@@ -40,29 +40,29 @@ public string GetNews()
...
+        {
+            var url = "http://www.cbr.ru/scripts/XML_daily.asp";
+
+            HttpWebRequest httpWebRequest = (HttpWebRequest)WebRequest.Create(url);
+            HttpWebResponse httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream);
+
+            var top5Valutes = new List<Valute>();
```
Comment (Cerberous007) [inline Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs:59]: The numbers in the name look strange

File: Services/BrokerService/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BrokerService
+{
+    public static class Constants
+    {
+        internal static int Depth = 5;
+        public static string Token = "example ";
```
Comment (Cerberous007) [inline Services/BrokerService/Constants.cs]: Why do you need this class? Why constant token is "example"?

Comment (denis-kriv) [reply Services/BrokerService/Constants.cs]: If the token changes, we can easily change it in one place, the same with the depth of the glass

Comment (denis-kriv) [reply Services/BrokerService/Constants.cs]: We will change the token when we recognize it

File: Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs (hunk 1 above)
Comment (denis-kriv) [reply Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs:59]: I did not change this file!!)

File: Services/BrokerService/Constants.cs (hunk 2 above)
Comment (nmamro) [reply Services/BrokerService/Constants.cs]: > We will change the token when we recognize it

Why not to use appsettings?

File: Services/BrokerService/BrokerService.csproj.BASE [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (nmamro) [inline Services/BrokerService/BrokerService.csproj.BASE]: Use git properly

File: Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs (hunk 1 above)
Comment (nmamro) [reply Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs:59]: > I did not change this file!!)

Then you better find out why it is here

File: Services/BrokerService/Controllers/BrokerController.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using BrokerServices;
+using BrokerServices.Utils;
+using Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BrokerService.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (ifelseelif) [inline Services/BrokerService/Controllers/BrokerController.cs]: why do you use api ?

**Comment by nmamro** [comment]
Guys, fix this request please, it's becoming too old

File: Services/BrokerService/BrokerService.csproj.BASE (hunk 3 above)
Comment (denis-kriv) [reply Services/BrokerService/BrokerService.csproj.BASE]: fixed

File: Services/BrokerService/Constants.cs (hunk 2 above)
Comment (denis-kriv) [reply Services/BrokerService/Constants.cs]: fixed

File: Services/BrokerService/Controllers/BrokerController.cs (hunk 4 above)
Comment (denis-kriv) [reply Services/BrokerService/Controllers/BrokerController.cs]: fixed

**Pull request closed without merging**